code/modules/crafting/craft.dm

```diff
@@ -290,7 +301,7 @@
 		qdel(DL)
 
 
-/datum/component/personal_crafting/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.not_incapacitated_turf_state)
+/datum/component/personal_crafting/ui_interact(x, mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.not_incapacitated_turf_state)
```

Comment:
These arguments are not what gets passed by the click signal. Filter through another proc if you wish.

Reply:
everything besides user are optional and were never used for crafting, thus i only care for user, x is the image i dont need